demos/snackbar.html

```diff
@@ -0,0 +1,90 @@
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500">
+    <script src="assets/material-design-lite.css.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <main>
+      <h1>MDL Snackbar</h1>
+      <section>
+        <h2>Basic Example</h2>
+        <div class="mdl-snackbar" id="mdl-js-snackbar">
+          <div class="mdl-snackbar__text"></div>
+          <button type="button" class="mdl-snackbar__action"></action>
+        </div>
+        <button type="button" id="show-snackbar">Show snackbar</button>
+        <button type="button" id="show-multiline">Show multiline</button>
+      </section>
+      <section dir="rtl">
+        <h2>RTL Example</h2>
+        <div class="mdl-snackbar" id="mdl-rtl-js-snackbar">
+          <div class="mdl-snackbar__text"></div>
+          <button type="button" class="mdl-snackbar__action"></action>
+        </div>
+        <button type="button" id="show-rtl-snackbar">Show snackbar</button>
+        <button type="button" id="show-rtl-multiline">Show multiline</button>
+      </section>
+    </main>
+    <script src="assets/material-design-lite.js" charset="utf-8"></script>
+    <script>
+      (function(global) {
+        'use strict';
+        var snackbar = new global.mdl.Snackbar(document.getElementById('mdl-js-snackbar'));
+        var rtlSnackbar = new global.mdl.Snackbar(document.getElementById('mdl-rtl-js-snackbar'));
+        document.getElementById('show-snackbar').addEventListener('click', function() {
+          snackbar.show({
+            message: 'Archived with a slightly longer message.',
+            actionText: 'Undo',
+            actionHandler: function() {
+              console.log('Undo it!');
+            }
+          });
+        });
+        document.getElementById('show-multiline').addEventListener('click', function() {
+          snackbar.show({
+            message: 'This item already has the label \'travel\'. You can add a new label. I really want this to be too long.',
+            multiline: true,
+            actionText: 'Long Action',
+            actionHandler: function() { console.log('Long Action'); }
+          });
+        });
+        document.getElementById('show-rtl-snackbar').addEventListener('click', function() {
+          rtlSnackbar.show({
+            message: 'Archived with a slightly longer message.',
+            actionText: 'Undo',
+            actionHandler: function() {
+              console.log('Undo it!');
+            }
+          });
+        });
+        document.getElementById('show-rtl-multiline').addEventListener('click', function() {
+          rtlSnackbar.show({
+            message: 'This item already has the label \'travel\'. You can add a new label. I really want this to be too long.',
+            multiline: true,
+            actionText: 'Long Action',
+            actionHandler: function() { console.log('Long Action'); }
+          });
+        });
+      })(this);
```

Comment:
Do you think it would be useful to show an example that omits an action? Like a status update?